.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should go out. You can configure this as a global ignore in your local for avoiding the problem in the future. Just put in ~.gitconfig these lines:
```
[core]
        excludesfile = ~/.gitignore
```